src/ledger-iframe-bridge.ts

```diff
@@ -85,11 +86,21 @@ type IFramePostMessage<TAction extends IFrameMessageAction> =
     target: typeof LEDGER_IFRAME_ID;
   };
 
-export class LedgerIframeBridge implements LedgerBridge {
+const BRIDGE_URL = 'https://metamask.github.io/eth-ledger-bridge-keyring';
+
+export type LedgerIframeBridgeOptions = {
+  bridgeUrl: string;
+};
+
+export class LedgerIframeBridge
+  implements LedgerBridge<LedgerIframeBridgeOptions>
+{
   iframe?: HTMLIFrameElement;
 
   iframeLoaded = false;
 
+  bridgeUrl = '';
+
```

Comment:
Porting my answer from the discussion on slack to not scatter information:

I see that currently in production we also deserialise the bridgeUrl along with the account-related stuff, so we currently mixup the bridge configuration with the keyring related values, but this doesn't take into account that the bridge might change in the future.

This was my consideration:
- If we serialise these bridge parameters, we expect them to be used when deserialising. This is fair when the configured bridge is always available, but what if in the future we need to move to another transport? We will probably have to migrate the user's serialised keyrings or add some condition to handle that in the keyring deserialise method, which probably will not be nice
- If the transport depends on the device where the keyring is being used, shouldn't the client (MM extension or mobile) decide how to instantiate the transport method, rather than loading it from persisted vault?

If we don't expect different users (with the same device) to use different transport methods (so we don't need to save the configuration on user storage), then I think we can safely remove the transport config from the keyring serialized data.

Reply:
totally agree